games-simulation/lincity-ng/lincity-ng-2.9_beta.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit eutils multiprocessing
```

Comment:
fast-glancing I don't see eutils being used in here.

Reply:
Right.